server/index.js

```diff
@@ -15,4 +20,9 @@ app.listen(PORT, () => {
   console.log(`App is listening at localhost:${PORT}`);
 });
 
+db.sync({ force: true })
```

Comment:
make sure you're catching an error. Also we'll need to write a conditional to make sure we don't { force: true } in our deployed app.

Not sure why the sync isn't working. Have you tried to console.log(db)? I run into this error quite often and it usually has to do with importing and exporting files.

Reply:
After changed the small syntax area from the " **server/db/database.js** "  to  `module.exports = db;` and changing each of the model file  to bring in `const db = require('./../database.js');` without the bracket notation   
 {db }. The database should be able to sync just fine afterwards

Reply:
Why don't we want { force: true }?

> make sure you're catching an error. Also we'll need to write a conditional to make sure we don't { force: true } in our deployed app.

Reply:
{ force: true } clears all the tables in a database. Something we DEFINITELY don't want to do in a deployed app.

Reply:
Ok, I'll leave a comment there.